bahmni-emr-api/pom.xml

```diff
@@ -68,6 +68,22 @@
             <groupId>org.bahmni.module</groupId>
             <artifactId>bahmni-commons-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.bahmni.module</groupId>
+            <artifactId>fhir-ts-services-int-api</artifactId>
+            <version>${terminologyServices.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
do we need direct dependency to fhir2 module? if fhir-ts-service-int-api is dependent on the fhir2 module, then explicit dependency is not required

Reply:
@angshu 
fhir2-api is **provided** dependency in fhir-ts-service-int-api and is not available as **provided** dependency in this module. [**Provided** dependency is not transitive](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope). 
However, compile time dependencies in fhir-ts-service-int-api module available as **provided** dependency in this module.